massgov/mayflower-patternlab
# DP-8645  remove nbsp from decorative links

## Description
Removes the non-breaking space from the decorative links and uses css to space correctly 

## Related Issue / Ticket

- [DP-8645](https://jira.state.ma.us/browse/DP-8645)

## Steps to Test

1.  Visit the Organization elected officials page  (/?p=pages-organization-elected-official)
2. Inspect one of the decorated links
3. You should see that the nbsp has been removed from the link

## Screenshots
<img width="988" alt="screen shot 2018-04-18 at 5 18 20 pm" src="https://user-images.githubusercontent.com/18662734/38958902-991e83d4-432c-11e8-8fd4-5d31d56ea940.png">

## Additional Notes:
There are a large number of backstop files included, because removing the space and refactoring the css caused some links to break on different words.
